MIAUI/ViewModels/SubtasksViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
```

Comment:
How about adding a `GlobalUsings.cs` file to the root and moving the common using statements there to clean up your other files. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#global-modifier